servers/src/test/java/tachyon/worker/block/BlockStoreMetaTest.java

```diff
@@ -0,0 +1,200 @@
+        MEM_QUOTA + "b");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, 1), "HDD");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, 1),
+        tachyonHome + "/disk1," + tachyonHome + "/disk2");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, 1),
+        HDD_DISK1_QUOTA + "," + HDD_DISK2_QUOTA);
+    
+    mMetadataManager = BlockMetadataManager.newBlockMetadataManager(tachyonConf);
+    
+    // Add and commit COMMITTED_BLOCKS_NUM temp blocks repeatedly
+    StorageDir dir = mMetadataManager.getTier(1).getDir(0);
+    for (long blockId = 0L; blockId < COMMITTED_BLOCKS_NUM; blockId ++) {
+      // prepare temp block
+      TempBlockMeta tempBlockMeta = new TempBlockMeta(TEST_USER_ID, blockId, TEST_BLOCK_SIZE, dir);
+      mMetadataManager.addTempBlockMeta(tempBlockMeta);
+
+      // write data
+      File tempFile = new File(tempBlockMeta.getPath());
+      
+      // create storage directory for user TEST_USER_ID at first loop
+      if (blockId == 0L) {
+        if (!tempFile.getParentFile().mkdir()) {
+          throw new IOException(String.format(
+              "Parent directory of %s can not be created for temp block", tempBlockMeta.getPath()));
+        }
+      }
+      
+      BlockWriter writer = new LocalFileBlockWriter(tempBlockMeta);
+      writer.append(BufferUtils.getIncreasingByteBuffer(Ints.checkedCast(TEST_BLOCK_SIZE)));
+      writer.close();
+
+      // commit block
+      Files.move(tempFile, new File(tempBlockMeta.getCommitPath()));
+      mMetadataManager.commitTempBlockMeta(tempBlockMeta);
+      
+    }
+    mBlockStoreMeta = new BlockStoreMeta(mMetadataManager);
+  }
+  
+  @Test
+  public void getBlockListTest() {
+    Map<Long, List<Long>> dirIdToBlockIds = new HashMap<Long, List<Long>>();
+    for (StorageTier tier : mMetadataManager.getTiers()) {
+      for (StorageDir dir : tier.getStorageDirs()) {
+        dirIdToBlockIds.put(dir.getStorageDirId(), dir.getBlockIds());
+      }
+    }
+    Map<Long, List<Long>> actual = mBlockStoreMeta.getBlockList();
+    Assert.assertEquals(TEST_DIR_NUM, actual.keySet().size());
+    Assert.assertEquals(dirIdToBlockIds, actual);
+  }
+  
+  @Test
+  public void getCapacityBytesTest() {
+    long expectedCapacityBytes = MEM_QUOTA + HDD_DISK1_QUOTA + HDD_DISK2_QUOTA;
+    Assert.assertEquals(expectedCapacityBytes, mBlockStoreMeta.getCapacityBytes());
+  }
+
+  @Test
+  public void getCapacityBytesOnDirsTest() {
+    Map<Long, Long> dirsToCapacityBytes = new HashMap<Long, Long>();
+    for (StorageTier tier : mMetadataManager.getTiers()) {
+      for (StorageDir dir : tier.getStorageDirs()) {
+        dirsToCapacityBytes.put(dir.getStorageDirId(), dir.getCapacityBytes());
+      }
+    }
+    Assert.assertEquals(dirsToCapacityBytes, mBlockStoreMeta.getCapacityBytesOnDirs());
+    Assert.assertTrue(mBlockStoreMeta.getCapacityBytesOnDirs().values()
```

Comment:
I think there should also be a size() check, in order to be equivalent to an equality check.